packages/providers/src.ts/base-provider.ts

```diff
@@ -609,13 +609,19 @@ export class BaseProvider extends Provider {
         const limit = 100;
         const oversizeResponseLength = limit + 1;
         let epContractsLogs = '/api/v1/contracts/' + params.filter.address + '/results/logs?limit=' + oversizeResponseLength;
+        if (params.filter.topics?.length > 0 && (fromTimestampFilter == "" || toTimestampFilter == "")) {
+            return logger.throwArgumentError(
+                "topic filter requires fromTimestamp/toTimestamp fields set",
+                Logger.errors.INVALID_ARGUMENT,
+                params.filter);
+        }
```

Comment:
Instead of throwing an error can we provide min and max amounts for `from` & `to` to hide the complexity to the users. ethers allows you to filter for topics without specifying `blocks` thus we should be able to do it for `timestamps` here if possible